generators/chaincode/templates/kotlin/.settings/org.eclipse.buildship.core.prefs

```diff
@@ -0,0 +1,13 @@
+arguments=
+auto.sync=false
+build.scans.enabled=false
+connection.gradle.distribution=GRADLE_DISTRIBUTION(WRAPPER)
+connection.project.dir=
+eclipse.preferences.version=1
+gradle.user.home=/home/matthew/.sdkman/candidates/gradle/current
```

Comment:
Don't think we want these .settings files in here, especially as they've got your home directory in 😉